scripts/main.js

```diff
@@ -9,7 +9,11 @@ class NameMaker {
 function initActorSheetHook(app, html, data) {
   let openBtn = $(`<a class="open-actor-name-maker" title="Make Name"><i class="fas fa-random"></i>Make Name</a>`);
   openBtn.click(ev => {
-    new MakerListWindow(app.document).render(true, {width: 480});
+    new MakerListWindow(app.document).render(true, {
```

Comment:
did this in a few places, just to let most of the windows be resizable, I tried to keep your defaults in place if you had any.